src/main/java/bio/terra/service/snapshot/SnapshotService.java

```diff
@@ -261,25 +261,24 @@ void validateForByRequestIdMode(SnapshotRequestContentsModel snapshotRequestCont
     if (snapshotRequestContents.getMode() != SnapshotRequestContentsModel.ModeEnum.BYREQUESTID) {
       return;
     }
-    SnapshotAccessRequestResponse snapshotAccessRequest =
+    SnapshotAccessRequestModel snapshotAccessRequest =
         snapshotRequestDao.getById(
             snapshotRequestContents.getRequestIdSpec().getSnapshotRequestId());
-    if (snapshotAccessRequest.getStatus() != SnapshotAccessRequestStatus.APPROVED) {
+    if (snapshotAccessRequest.status() != SnapshotAccessRequestStatus.APPROVED) {
       throw new ValidationException(
           "Snapshot request must be approved before creating a snapshot.");
     }
-    if (snapshotAccessRequest.getCreatedSnapshotId() != null) {
+    if (snapshotAccessRequest.createdSnapshotId() != null) {
       throw new ValidationException(
           "Snapshot with id %s is already created from request with id %s"
-              .formatted(
-                  snapshotAccessRequest.getCreatedSnapshotId(), snapshotAccessRequest.getId()));
+              .formatted(snapshotAccessRequest.createdSnapshotId(), snapshotAccessRequest.id()));
     }
-    String flightId = snapshotAccessRequest.getFlightid();
+    String flightId = snapshotAccessRequest.flightid();
     if (flightId != null && jobService.unauthRetrieveJobState(flightId) != FlightStatus.ERROR) {
       throw new ValidationException(
           "Snapshot Create Flight with id %s is still running".formatted(flightId));
     }
-    var requesterEmail = snapshotAccessRequest.getCreatedBy();
+    var requesterEmail = snapshotAccessRequest.createdBy();
     if (requesterEmail == null || !ValidationUtils.isValidEmail(requesterEmail)) {
```

Comment:
just curious, why move from the getter to accessing the field directly?

Reply:
That's the default syntax for records, I believe?